README.md

```diff
@@ -162,7 +162,7 @@ Here is a rundown of other arguments for your docker-compose / docker run.
 - Docker's default network mode `bridge` isolates the container from the host's network. This is a more secure setting, but requires setting the Pi-hole DNS option for _Interface listening behavior_ to "Listen on all interfaces, permit all origins".
 - If you're using a Red Hat based distribution with an SELinux Enforcing policy, add `:z` to line with volumes.
 
-### Installing on Ubuntu or Fedora
+## Installing on Ubuntu or Debian
```

Comment:
You changed the title, but the text below is related to Ubuntu (17.10+) and Fedora (33+).

Reply:
Good catch, my mistake. I've corrected it.